api/src/db/migrations/versions/2023_10_18_basic_opportunity_table.py

```diff
@@ -0,0 +1,48 @@
+"""Basic opportunity table
+
+Revision ID: fdd9312633d8
+Revises: 4ff1160282d1
+Create Date: 2023-10-18 16:37:18.252136
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "fdd9312633d8"
+down_revision = "4ff1160282d1"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Still learning to navigate all this, but an alembic migration will run with our deploys of this? It's an actual create table.. if I'm understanding correctly. So that we don't conflict with DMS replication, we should probably avoid any migrations for now and disable those.

Reply:
We'd want to disable those in the CI/CD, the migration files are still needed locally for setting up your local DB. Let me take a look at whatever is in the CI/CD to see what it is doing.

Reply:
Looks like the deploy workflow calls database-migrations: https://github.com/HHS/simpler-grants-gov/blob/main/.github/workflows/deploy.yml which actually does build-and-publish + migrate. Could just modify the deploy workflow to call build-and-publish directly and remove the database migration one entirely.